### ISSUE TITLE
attempt to access invalid path

### ISSUE THREAD

**Issue by AlphaDog1** [kolton/d2bot-with-kolbot#1333]
First off i have three folders full of the same stuff. Branches, Tags and Trunk. I Am assuming to use 1.13dcore15 if this is correct then i keep getting attempt to access invalid address. if im to use Trunk then d2bot wont open atoll. Help please

**Comment by DarkHorseDre** [comment]
download the latest and use master:
https://github.com/kolton/d2bot-with-kolbot

**Comment by AlphaDog1** [comment]
that is the one i downloaded and it dosent have a master folder only, trunk branches and tags

**Comment by DarkHorseDre** [comment]
yeah trunk = master.  they merged recently - just download that version which is standard and default.  1.13 is the older d2 version from ages ago, as per the screenshot below.
![image](https://user-images.githubusercontent.com/41862641/54295499-04935780-45ab-11e9-8f95-c6c8ec8b9bb5.png)

**Comment by AlphaDog1** [comment]
ok then this brings me to the real issue then, i click on d2bot in master folder and it dosent even open for me

**Comment by AlphaDog1** [comment]
ok sry nvm that top reply is void

**Comment by AlphaDog1** [comment]
i go into branches there is two folders 113d and 113d core 15

**Comment by AlphaDog1** [comment]
in both are the same files step one now is to figure out what folder is the one i use, then each one has its own problems when i go to push start so when you tell me the proper folder i will tell you the error. first thanks a ton for taking the time you are a great person for being so helpful

**Comment by DarkHorseDre** [comment]
no you shouldnt go into the 113 folder - I dont even have them

just look at the path to d2bs here, as above:
https://github.com/kolton/d2bot-with-kolbot

the d2bs you need is in that folder. the 113 branches are not referenced in this path.  when you download, this is the path you need to follow.

**Comment by AlphaDog1** [comment]
im sorry to sound so stupid, problem is everytime i click clone or download i copy then checkout in a new folder i get the branch, tags and trunk folders in my branch folder all that is there is 113d or 113d core 15. am i do download a way i am not comperhending

**Comment by AlphaDog1** [comment]
i8 realize now i am missing the whole master folder

**Comment by AlphaDog1** [comment]
![folders](https://user-images.githubusercontent.com/48522710/54298294-dbfd6480-458e-11e9-934c-88a6fba9ba11.png)
![whats in branches](https://user-images.githubusercontent.com/48522710/54298300-ddc72800-458e-11e9-80ad-132d7351a918.png)
![whats in tags](https://user-images.githubusercontent.com/48522710/54298301-df90eb80-458e-11e9-8aa1-409768e026ee.png)
![whats in trunks](https://user-images.githubusercontent.com/48522710/54298304-e0298200-458e-11e9-9b1c-29fb1954b707.png)

**Comment by AlphaDog1** [comment]
this is an image of my entire kolbot folder that i get after svn checkout and update

**Comment by AlphaDog1** [comment]
your help on getting this working is greatly appriciated

**Comment by AlphaDog1** [comment]
if i am in fact missing anything important file wise please help me figure out how to get it and add it into my folder

**Comment by DarkHorseDre** [comment]
the last folder image you sent is the one you need.

Trunk is always the main branch (release version I guess) - it was merged recently but ultimately you always download trunk/master.

**Comment by AlphaDog1** [comment]
ok so when i click run as admin i click on it but it wont open or give any sign it wants to open

**Comment by DarkHorseDre** [comment]
Disable your antivirus and firewall temporarily maybe?

**Comment by AlphaDog1** [comment]
nope

**Comment by AlphaDog1** [comment]
im not supose to move anything anywhere? i litterly just download it and double click on d2bot.exe and its supose to work

**Comment by DarkHorseDre** [comment]
yep

**Comment by DarkHorseDre** [comment]
fyi when you select the master branch as per the above posts, download the repo and master is then included in the path - I just did it from work so cant open the d2bs:
![image](https://user-images.githubusercontent.com/41862641/54299211-512e6100-45b2-11e9-9ace-c226f7607796.png)

**Comment by DarkHorseDre** [comment]
Stop creating new topics for this.

ensure you have installed everything required:
https://github.com/kolton/d2bot-with-kolbot/wiki

**Comment by AlphaDog1** [comment]
if i download as a zip file thats ok? because the only way i can download with master included is by unzipping with winzip. i thought i had to make sure to use svn

**Comment by AlphaDog1** [comment]
i got the same thing you got now it just still wont open for me

**Comment by ghostwolf1228** [comment]
@DarkHorseDre issue still persists, even when everything has been installed and the setup guide followed. 
![image](https://user-images.githubusercontent.com/5233887/63228307-cc212680-c1a5-11e9-9294-88b341273ee6.png)

**Comment by Aye54** [comment]
found a solution to the problem?

**Comment by Embed-Master** [comment]
I also got this problem. I use master branch, tried any settings/versions/compatibility modes... Anything was useless.

**Comment by nicthegarden** [comment]
Same issue here... Did not change anything in my setup
